### PR TITLE
Add docs about how to build eStargz

### PR DESCRIPTION
**Pull request by ktock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,31 +128,59 @@ This section describes some of them.
 
 You can try our pre-converted eStargz images on ghcr.io listed in [Trying pre-converted images](/docs/pre-converted-images.md).
 
-### Registry-side conversion with `estargz.kontain.me`
+### Building using BuildKit
```
Comment (AkihiroSuda) [inline README.md:131]:
```suggestion
### Building eStargz images using BuildKit
```

Comment (ktock) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -128,31 +128,59 @@ This section describes some of them.
...
 
-You can convert arbitrary images into eStargz on the registry-side, using [`estargz.kontain.me`](https://estargz.kontain.me).
-`estargz.kontain.me/[image]` serves eStargz-converted version of an arbitrary public image.
+BuildKit supports building eStargz image since v0.10.
 
-For example, the following Kubernetes manifest performs lazy pulling of eStargz-formatted version of `docker.io/library/nginx:1.21.1` that is converted by `estargz.kontain.me`.
+You can try it using [Docker Buildx](https://docs.docker.com/buildx/working-with-buildx/).
```
[the remark this answers is not in the document]
Comment (ktock) [reply README.md:135]: Added version informaion.

File: README.md [hunk 3]
```diff
@@ -128,31 +128,59 @@ This section describes some of them.
...
-    - containerPort: 80
+```
+$ docker buildx build -t ghcr.io/ktock/hello:esgz \
+    -o type=registry,oci-mediatypes=true,compression=estargz,force-compression=true \
+    /tmp/buildctx/
 ```
 
-> WARNING: Before trying this method, read [caveats from kontain.me](https://github.com/imjasonh/kontain.me#caveats). If you rely on it in production, you should copy the image to your own registry or build eStargz by your own using `ctr-remote` as described in the following.
+> NOTE: `force-compression=true` isn't needed if the base image is already eStargz.
+
```
[the remark this answers is not in the document]
Comment (ktock) [reply README.md:148]: added the note.

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**